### DISCUSSION TITLE
copy/paste paths in the address bar

### DISCUSSION THREAD

**Discussion by morpholino** [iterate-ch/cyberduck#15592] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
One reason why I never really started using CyberDuck is that I cannot copy or paste paths in the address bar, which is something I use frequently in FileZilla to help me navigate directories in my CLI. Also, when a symlink is followed in CyberDuck, it is not followed physically (as in cd -P), making it harder to navigate complex structures (e.g., when I have a symlink to a storage node in my home directory, it might not be accessible for other users). 

**Describe the solution you'd like**
Adding the option to copy/paste paths in the address bar.

**Describe alternatives you've considered**
Alternatively, a terminal console window that mirrors the file viewer window would be even better. It is possible to open the current path in the terminal, but changing directories is not synced afterwards.

**Comment by AliveDevil** [comment, marked as answer]
There is `Go to -> Go to folder` (Ctrl-G) - which got copied/applied from the Cyberduck macOS interface, where macOS uses `⇧⌘G` in Finder for the prompt.

You can select `Go to -> Open in PuTTy` to open an interactive SSH shell.

**Reply by cocokane** [reply]
This still does not help me copy the path from the address bar. The closest thing I can do is copy a file (using cmd C) in that directory, and paste it elsewhere, and then backspace some values cyberduck adds to the file path. This is a rather inconvenient method, and simply being able to copy the path from the address bar would be a great addition.

**Reply by morpholino** [reply]
cmd+G does the job for me. pulls up the full path to the directory currently showing
